shared/charts/couchdb-prometheus-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  namespace: {{ .Release.Namespace | quote }}
+  name: {{ template "couchdb_prometheus_exporter.name" . }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: couchdb-prometheus-exporter
+    spec:
+      containers:
+      - name: couchdb-prometheus-exporter
+        image: "{{ .Values.image.repository }}@{{ .Values.image.checksum }}"
+        args:
+        - -logtostderr
+        - -couchdb.uri=$(COUCHDB_URI)
+        - -couchdb.username=$(COUCHDB_USERNAME)
+        - -couchdb.password=$(COUCHDB_PASSWORD)
+        - -databases=$(COUCHDB_DATABASES)
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        ports:
+        - containerPort: {{ .Values.exporter_listen_port }}
+        env:
+        - name: COUCHDB_URI
+          value: {{ .Values.couchdb_uri }}
+        - name: COUCHDB_USERNAME
+          value: {{ .Values.couchdb_username }}
+        - name: COUCHDB_PASSWORD
+          value: {{ .Values.couchdb_password }}
+        - name: COUCHDB_DATABASES
+          value: {{ .Values.couchdb_databases }}
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
+      - name: prometheus-to-sd
+        image: gcr.io/google-containers/prometheus-to-sd:v0.3.2
```

Comment:
I think this would be pretty good to parametrize, especially the image version. Sorry I didn't notice before.

Reply:
Done.